coffee-chats/src/test/java/com/google/step/coffee/data/CalendarUtilsTest.java

```diff
@@ -24,17 +24,38 @@
 public class CalendarUtilsTest extends TestHelper {
 
   private final String testUser1Id = "1111";
-  private final String testUser2Id = "2222";
-  private final String testUser3Id = "3333";
 
   @Test
   public void eventDescriptionContainsCommonTags() {
     final TimeSlot timeslot = new TimeSlot(ZonedDateTime.now(), Duration.ofMinutes(30));
     final List<String> participantIds = new ArrayList<>();
     final List<String> commonTags = new ArrayList<>();
 
-    setUpEventInfo(participantIds, commonTags);
-    addTestUsersInfo();
+    final String testUser2Id = "2222";
+    final String testUser3Id = "3333";
+
+    participantIds.add(testUser1Id);
+    participantIds.add(testUser2Id);
+    participantIds.add(testUser3Id);
+
+    commonTags.add("Football");
+    commonTags.add("Photography");
+
+    User testUser1 = mock(User.class);
+    User testUser2 = mock(User.class);
+    User testUser3 = mock(User.class);
+
+    when(testUser1.getUserId()).thenReturn(testUser1Id);
+    when(testUser1.getEmail()).thenReturn("testUser1@example.com");
+    when(testUser2.getUserId()).thenReturn(testUser2Id);
+    when(testUser2.getEmail()).thenReturn("testUser2@example.com");
+    when(testUser3.getUserId()).thenReturn(testUser3Id);
+    when(testUser3.getEmail()).thenReturn("testUser3@example.com");
+
+    UserStore userStore = new UserStore();
```

Comment:
Please add a comment explaining how this works despite the variable being apparently unused